README.md

```diff
@@ -66,16 +66,19 @@ class PreScale(Transformation):
 
     def run(self):
         for env in self.environments:
-            file = self.repo.get_objects(f'overlays/{self.environments}/envconfig-values.yaml')
+            objects = self.repo.get_objects(f'overlays/{env}/envconfig-values.yaml')
 
-            for obj in file:
+            min_replicas = None
+            for obj in objects:
                 if obj['kind'] != 'HorizontalPodAutoscaler':
                     continue
 
                 if obj['spec']['minReplicas'] != obj['spec']['maxReplicas']:
+                    min_replicas = obj['spec']['minReplicas']
                     obj['spec']['maxReplicas'] = obj['spec']['minReplicas']
```

Comment:
Should this be set to `min_replicas` then?

Reply:
👍